docs/audit/WAVE0_KG_RELATIONS_FINDINGS.md

```
@@ -0,0 +1,389 @@
+
+-- But edges table still has:
+SELECT * FROM knowledge_graph_edges;
+-- Returns: [
+--   {source_id: node-1, target_id: node-2},  ← ORPHANED!
+--   {source_id: node-2, target_id: node-3},  ← OK
+-- ]
+```
+
+**Fix**: Add cascading cleanup in `DeduplicateGraphUseCase`
+(Note: GraphPersistencePort does not have a `deleteEdges` method; use `persistKnowledgeGraph` with delete+insert semantics or add this method to the port interface)
+```typescript
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)